spec/requests/data_api/tad_degree_class_export_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'GET /data-api/applications-by-subject-route-and-degree-grade/latest', type: :request, sidekiq: true do
+  include DataAPISpecHelper
+
+  it_behaves_like 'a TAD API endpoint', '/latest'
+
+  it 'returns the latest tad degree class export' do
+    drama = create(:subject, code: '13')
+    first_application_form = create(:completed_application_form)
+    create(:application_qualification, level: 'degree', grade: 'Upper second-class honours (2:1)', application_form: first_application_form)
+    scitt_provider = create(:provider, provider_type: 'scitt')
+    first_course = create(:course, provider: scitt_provider, subjects: [drama])
+    first_course_option = create(:course_option, course: first_course)
+
+    create(:application_choice, :with_declined_offer, course_option: first_course_option, application_form: first_application_form)
+
+    data_export = DataExport.create!(
+      name: 'Weekly export of the TAD degree class export',
+      export_type: :tad_degree_class_by_subject_and_outcome,
+    )
+    DataExporter.perform_async(SupportInterface::TADDegreeClassExport, data_export.id)
```

Comment:
And we might as well rename the export itself?

Reply:
yes! – fine to do the changes in here rather than the other PR? (Although will merge them together)